Formula/mysql@5.7.rb

```diff
@@ -100,10 +100,17 @@ def caveats
       To connect run:
           mysql -uroot
     EOS
+    if Dir.exist?("/usr/local/var/mysql/")
```

Comment:
Is it specifically `/usr/local/var/mysql/` or relative to the Homebrew prefix (i.e.): 

```suggestion
    if data = (var/"mysql") && data.exist?
```